django/db/backends/sqlite3/schema.py

```diff
@@ -11,26 +11,20 @@
 class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
 
     sql_delete_table = "DROP TABLE %(table)s"
-    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s)"
+    sql_create_fk = None
+    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s) DEFERRABLE INITIALLY DEFERRED"
```

Comment:
Related to what charettes asked, but asked in a slightly different way:

Assumption: basic migrations will handle the table rebuild, like existing sqlite schema changes. 

Question: if PRAGMA foreign_keys is going to be default on in django 2, how will the user "opt in" their old tables to a migration, without there being a difference in their model params?